mail/mailutils/Portfile

```diff
@@ -82,15 +83,28 @@ variant tcp_wrappers description {Build with tcp_wrappers support} {
     depends_build-append    port:tcp_wrappers
 }
 
-variant python27 description {Enable support for Python 2.7 scripting} {
-    configure.args-replace  --disable-python --enable-python
-    configure.python        ${prefix}/bin/python2.7
-    depends_lib-append      port:python27
-}
-
-variant gdbm conflicts description {Build with GNU DBM support} {
+variant gdbm description {Build with GNU DBM support} {
     configure.args-replace  --without-gdbm --with-gdbm
     depends_lib-append      port:gdbm
 }
 
-default_variants +ldap +tcp_wrappers
+variant ndbm description {Build with NDBM support} {
+    configure.args-replace  --without-ndbm --with-ndbm
+}
+
+variant gssapi description {Enable gssapi} {
+    configure.args-replace --without-gssapi --with-gssapi
```

Comment:
this doesn't conform to the modeline; the ``--without-gssapi` should be at column 29 (indentation with spaces and multiples of four. Please check that in the other variants you added as well.

Reply:
"should be" fixed.
thank you very much for your help and support!